### ISSUE TITLE
terra-application-navigation not themed for clinical-lowlight-theme

### ISSUE THREAD

**Issue by Me999999999** [cerner/terra-application#133]
# Bug Report

## Description

When trying to view terra-application-navigation in clinical-lowlight-theme the theme variables are not being correctly applied.
https://pages.github.cerner.com/orion/orion-application/components/orion-application/demo
<img width="1278" alt="Screen Shot 2021-02-11 at 1 13 59 PM" src="https://user-images.githubusercontent.com/23201118/107686640-1be19080-6c6b-11eb-82cb-c0c55610204f.png">

## Steps to Reproduce

1. Navigate to https://pages.github.cerner.com/orion/orion-application/components/orion-application/demo
2. Switch to clinical-lowlight-theme
3. Nav Bar and portions of the Utility Menu are not themed correctly

## Additional Context / Screenshots

<img width="1278" alt="Screen Shot 2021-02-11 at 1 13 59 PM" src="https://user-images.githubusercontent.com/23201118/107686640-1be19080-6c6b-11eb-82cb-c0c55610204f.png">

## Expected Behavior

All components should be correctly themed when using a theme.

## @ Mentions

@StephenEsser @mjhenkes

**Comment by neilpfeiffer** [comment]
Transferring issue to [Terra-Framework](https://github.com/cerner/terra-framework/issues).
